roles/analysis/tasks/analysis-c2r.yml

```diff
@@ -0,0 +1,40 @@
+---
+# TODO: Install convert2rhel for different OS?
+- name: Install packages for pre-conversion analysis on CentOS 7
+  ansible.builtin.package:
+    name: "{{ analysis_packages_centos7 }}"
+    # enablerepo: "{{ analysis_repos_centos7 }}" TODO?
+    state: latest
+  when: ansible_distribution == "CentOS" and ansible_distribution_major_version|int == 7
```

Comment:
As mentioned in a previous comment, I think we could delegate this job to the common role and reuse it in the conversion task as well